### PR TITLE
feature/CDAP-19521-remove-registry

### PR DESCRIPTION
**Pull request by de-lan**
This PR is to remove the unnecessary table registry from PostgreSqlStructuredTableAdmin. Instead, we use an in-memory LoadingCache to cache the table information.

### CODE REVIEW

**Review by fernst — APPROVED** [review]
LGTM

File: cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java [hunk 1]
```diff
@@ -277,13 +277,13 @@ public CloseableIterator<StructuredRow> multiScan(Collection<Range> keyRanges,
...
    * E.g.
-   *
+   * <p>
    * SELECT * FROM table WHERE key1 in (?,?) AND key2 in (?,?)
    * OR ((key3 >= ?) AND (key3 <= ?)) OR ((key4 >= ?) AND (key4 <= ?)) LIMIT limit
    *
    * @param keyFields a map from field name to field values that the query has to match with
-   * @param ranges the list of ranges to scan
-   * @param limit number of result
+   * @param ranges    the list of ranges to scan
```
Comment (chtyim) [inline cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java:285]: unnecessary whitespace change.

**Review by chtyim — APPROVED** [review]

File: cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java (hunk 1 above)
Comment (de-lan) [reply cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java:285]: I imported the [coding standard ](https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard)and it re-formatted this file. I guess it increased a bit readability? Since in IntelliJ it highlights the param and de-highlight the description:
image
